pkg/dbutil/common.go

```diff
@@ -421,3 +604,15 @@ func TableName(schema, table string) string {
 func escapeName(name string) string {
 	return strings.Replace(name, "`", "``", -1)
 }
+
+// ReplacePlaceholder will use args to replace '?', used for log.
+// tips: make sure the num of "?" is same with len(args)
+func ReplacePlaceholder(str string, args []string) string {
+	/*
+		for example:
+		str is "a > ? AND a < ?", args is {'1', '2'},
+		this function will return "a > '1' AND a < '2'"
+	*/
+	newStr := strings.Replace(str, "?", "'%s'", -1)
+	return fmt.Sprintf(newStr, utils.StringsToInterfaces(args)...)
```

Comment:
why need `StringsToInterfaces`?

Reply:
if use args will return error `cannot use args (type []string) as type []interface {} in argument to fmt.Sprintf`